addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -123,3 +123,21 @@ test("create sub thread from sub-thread list", async () => {
     await click("button[aria-label='Create Thread']");
     await contains(".o-mail-Discuss-threadName", { value: "MyEpicThread" });
 });
+
+test("'Threads' menu also available in a Thread", async () => {
```

Comment:
Name is confusing IMO. I'd rename it to better convey its purpose. Maybe `'Thread' menu available in sub channels` ?

Reply:
"sub channel" is technical name that has been chosen just because we didn't want to rename `thread` everywhere.
For end-users, they are "Thread", we don't name other conversations "Thread" so this is not misleading.

I much prefer we name test and steps as closely to UX as possible, so I'd not use "sub channels" wording here

Reply:
Well it has been chosen not to confuse us, who reads, write and maintain tests? Us. So to me sub-channel would be clearer.